### ISSUE TITLE
InvalidTypeIdException on getVoiceClient().listCalls()

### ISSUE THREAD

**Issue by jepsar** [Vonage/vonage-java-sdk#443]
**Describe the bug**
Exception when trying to get listCalls()

**To Reproduce**
Try to execute getVoiceClient().listCalls()

**Expected behavior**
No exception

**Vonage SDK Version**
7.2.0

**Java SDK Version**
openjdk version "17.0.1" 2021-10-19 
OpenJDK Runtime Environment Temurin-17.0.1+12 (build 17.0.1+12) 
OpenJDK 64-Bit Server VM Temurin-17.0.1+12 (build 17.0.1+12, mixed mode)

**Additional context/ Error Log**
```
Caused by: com.fasterxml.jackson.databind.exc.InvalidTypeIdException: 
Could not resolve type id 'app' as a subtype of `com.vonage.client.voice.Endpoint`: known type ids = [phone, sip, websocket] (for POJO property 'from')
 at [Source: (String)"{"page_size":10,"record_index":0,"count":63,"_embedded":{"calls":[{"uuid":"xxx","status":"cancelled","direction":"inbound","rate":"0.00450000","price":"0.00000000","duration":"0","network":"US-VIRTUAL-BANDWIDTH","conversation_uuid":"xxx","start_time":"","end_time":"","to":{"type":"phone","number":"xxx"},"from":{"type":"phone","number":"xxx"},"_links":{"self":{"href":"/v1/calls/xxx"}}},{"[truncated 4081 chars]; line: 1, column: 4273] (through reference chain: com.vonage.client.voice.CallInfoPage["_embedded"]->com.vonage.client.voice.EmbeddedCalls["calls"]->java.lang.Object[][9]->com.vonage.client.voice.CallInfo["from"])
```

**Comment by jepsar** [comment]
Same happens when I try to get `voiceClient.getCallDetails(callId)`.

**Comment by jepsar** [comment]
We are using "In-App Voice". Maybe that's related?

**Comment by SMadani** [comment]
Thank you for reporting this and submitting a PR! I will add this fix to the v7.3.0 release close this issue once it's done.
Out of curiosity, how did you encounter this? I presume the call was made from another SDK or the REST API? I'm just trying to create a minimal reproducible example which would help with testing this in practice.

**Comment by SMadani** [comment]
Should be fixed in #446.
